tools/emptyDropsWrapper/test/test_emptyDropsWrapper.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+## File: test_emptyDropsWrapper.sh
+## Author: Nikolas Barkas
+## Date: January 3, 2019
+## Description: Unit test for emptyDropsWrapper script
+
+## Download some sample data from 10X
+printf "Downloading pbmc4k data..."
+wget -q http://cf.10xgenomics.com/samples/cell-exp/2.1.0/pbmc4k/pbmc4k_raw_gene_bc_matrices.tar.gz
```

Comment:
I suggest going through this script and replacing any hard-coded info with variables so that you could easily test this with other datasets.

Reply:
Ok I'll improve this, but I don't think it can be completely automated because the input data structure might differ between datasets

Reply:
Done. There is one step in the cleanup that I am not sure how to implement best. I've put a note there. Suggestions welcome. Please check and let me know if this is ok now.

Reply:
I've addressed this now as discussed with @samanehsan